var/spack/repos/builtin/packages/jags/package.py

```diff
@@ -35,6 +35,7 @@ class Jags(AutotoolsPackage):
     homepage = "http://mcmc-jags.sourceforge.net/"
     url = "https://downloads.sourceforge.net/project/mcmc-jags/JAGS/4.x/Source/JAGS-4.2.0.tar.gz"
 
+	version('4.3.0', 'd88dff326603deee39ce7fa4234c5a43')
```

Comment:
whitespacing. Flake8 won't be happy ;)

Reply:
change approved. Please pull!

Reply:
You need to make the change on your branch and push it to your repo for GitHub to pick it up.